lib/ace/ext/modelist.js

```diff
@@ -167,7 +167,8 @@ var supportedModes = {
     XQuery:      ["xq"],
     YAML:        ["yaml|yml"],
     // Add the missing mode "Django" to ext-modelist
-    Django:      ["html"]
+    Django:      ["html"],
+	Razor:		 ["razor"]
```

Comment:
this is supposed to be a list of extensions for the mode, cshtml and maybe some other extensions.